civis/ml/_model.py

```diff
@@ -631,7 +643,10 @@ def __init__(self, model, dependent_variable,
                  cross_validation_parameters=None, model_name=None,
                  calibration=None, excluded_columns=None, client=None,
                  cpu_requested=None, memory_requested=None,
-                 disk_requested=None, verbose=False):
+                 disk_requested=None, notification_urls=None,
+                 success_email_subject=None, success_email_body=None,
+                 success_email_addresses=None, failure_email_addresses=None,
+                 stall_warning_minutes=None, verbose=False):
```

Comment:
I thought about this but we don't do it for required resources and additionally I worry the user will just have to look up the docs without this. Happy to have to change if you'd rather but this seems like it would be both easier to use and is currently in line with convention.